woocommerce/woocommerce-ios
# My Store Analytics: Custom Range UI for Visitors and Conversions

## Description


Closes #11972

For Custom Range tab, we want to redact information about Visitors and Conversions, because there is no accurate data for visits for a custom date range. However, we can show those information when an individual value in the chart is selected, since the data is accurate.


This PR also adds a new redaction view for those two labels, with an information icon. It will serve as starting point to add the bottom sheet information in task #12162 

## Testing instructions


1. Start app, create a custom range tab if you haven't,
2. In the custom range, ensure that Visitors and Conversions label are redacted with an information icon added,
3. Move to the other stats tabs, ensure that Visitors and Conversions are loaded properly.
4. Go back to custom range, select a value in the range, ensure that the Visitors and Conversions data are now shown,
5. Deselect the value, ensure the values are redacted again.

## Video


https://github.com/woocommerce/woocommerce-ios/assets/266376/f5bb63c5-e1f6-41e2-87b6-f5978664530d



---
